### DISCUSSION TITLE
Spotfire通过MYSQL connector连接Doris出现limit问题

### DISCUSSION THREAD

**Discussion by jarvan112** [apache/doris#15162] [A - General / Q&A]
问题描述：Spotfire通过MYSQL connector连接Doris查询一张有18万笔数据的表(t)时会在sql的后面自动加limit。比如我在spotfire中下的sql是select * from t ,但是到doris这边sql分成了两段 分别是select * from t limit 0,100000；select * from t limit 100000,100000。造成spotfire这边只能看到前10万笔数据。这边有在联系Spotfire的厂商这个问题的解法，同时也咨询Doris这边是否可以提供对应的解法

希望解决的是：
1、Doris是否有limit 100000的限制，如果有该如何解
2、Doris这边是否有配置可以去掉limit的功能，比如sql是  select * from t limit 0,100000   但在执行的时候只需执行select * from t即可。我这边业务的sql是不需要limit的

**Comment by LemonLiTree** [comment]
1，doris没用limit 10000的限制
2，也没用配置可以去掉limit，如果是在spotfire 里被加上了limit，需要在spotfire想办法去掉
